google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -898,10 +898,6 @@ def default_acl
         # @see https://cloud.google.com/storage/docs/json_api/v1/buckets/setIamPolicy
         #   Buckets: setIamPolicy
         #
-        # @param [Boolean] force Force load the latest policy when `true`.
```

Comment:
In response to the question raised in [issue 1430](https://github.com/GoogleCloudPlatform/google-cloud-ruby/issues/1430). I think we should leave the parameter in the documentation, because it's able to refresh the `Bucket` memoized `Policy` that `Policy#reload!` doesn't. I'm not sure if there's another use-case that I'm not aware of, but that for sure is important. 

@quartzmo  I'm not sure how to proceed while `Policy#reload!` doesn't update the `Policy` instance of `Bucket`. @remi and @dazuma, what are your thoughts on having `Policy#reload` returning an updated `Policy` but not updating the `Policy` instance in `Bucket`? Example shown below. 

> Quoting @quartzmo 
```ruby
bucket = storage.bucket "my-todo-app"

policy = bucket.policy # API call
policy_2 = bucket.policy.reload! # API call
policy_3 = bucket.policy # No API call. Stale reference, same instance as first line.
```